CMakeLists.txt

```diff
@@ -141,6 +141,10 @@ if(SFML_OS_ANDROID)
     set(CMAKE_CXX_CREATE_SHARED_LIBRARY_WITHOUT_STL "<CMAKE_CXX_COMPILER> <CMAKE_SHARED_LIBRARY_CXX_FLAGS> <LANGUAGE_COMPILE_FLAGS> <LINK_FLAGS> <CMAKE_SHARED_LIBRARY_CREATE_CXX_FLAGS> <SONAME_FLAG><TARGET_SONAME> -o <TARGET> <OBJECTS> <LINK_LIBRARIES>")
 endif()
 
+# Output variables to the config header
+configure_file(include/SFML/Config.hpp.in configured/include/SFML/Config.hpp @ONLY)
```

Comment:
Any particular you reason you picked BINARY_DIR/configured/include to put this code instead of something like BINARY_DIR/include? Does that extra nested directory buy us something?

Reply:
`configured` here serves as a prefix here that houses the configured source(s). I could omit `include` once the install rules are fixed, but that's for another PR.

Reply:
> once the install rules are fixed

What's broken about the install rules?

Reply:
`DESTINATION .` for one. Very unconventional.

Reply:
In my opinion it's reasonable to fix those install rules if it means making this PR simpler.

Reply:
Install rules are on the to-do list. They will be handled in a later PR. I'll resolve this.